docs/advanced/development/kubernetes.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Kubernetes
+tags:
+  - Developer
+  - Technical documentation
+  - Code
+  - Programming
+sidebar_position: 1
+---
+
+This guide provides step-by-step instructions for setting up a Kubernetes cluster using kind, configuring multiple nodes, and installing the Metrics Server.
+
+## Prerequisites
+
+Ensure you have the following installed:
+
+- [Docker](https://docs.docker.com/get-started/get-docker/) (required for running `kind` clusters).
+- [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/) (Kubernetes in Docker).
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/) (Kubernetes command-line tool).
+
+Additionally, ensure that Kubernetes tools are enabled in your environment by setting the following variable in your `.env` file:
+
+```bash
+# Enable Kubernetes tool
+ENABLE_KUBERNETES=true
+```
```

Comment:
This only applies to development. I would simply mention that they need to set the env